src/BundleAnalyzerPlugin.js

```diff
@@ -55,14 +55,21 @@ class BundleAnalyzerPlugin {
 
       if (actions.length) {
         // Making analyzer logs to be after all webpack logs in the console
-        setImmediate(() => {
-          actions.forEach(action => action());
+        setImmediate(async () => {
+          try {
+            for (let i = 0; i < actions.length; ++i) await actions[i]();
```

Comment:
Those actions are not related to each other and can be run concurrently. So it can be replaced with `await Promise.all(actions.map(action => action())`.